LDAR_Sim/src/initialization/leaks.py

```diff
@@ -52,7 +52,7 @@ def generate_leak(program, site, start_date, leak_count, days_active=0):
         'rate': leak_rate,
         'lat': float(site['lat']),
         'lon': float(site['lon']),
-        'status': 'active',
+        'status': 'untagged',
```

Comment:
This is more semantically accurate, good switch.

Reply:
Wait - is it? Isn't this why we have tagged = True/False? This variable is supposed to show whether active or repaired... untaggged status is redundant with "tagged" variable no?

Reply:
NO, because the repaired vs unrepaired is actually handled by the sites leak group now so it is not necessary. 

site['active_leaks] = [leak1:{status:'untagged}, leak2:{status:'tagged}]
site['repaired_leaks'] = [leak1:{status:'repaired'}]

tagged variable is / can be removed I will check.